release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -664,6 +664,11 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-15-removed-features"]
 === Removed features
 
+[id="ocp-4-14-openshift-default-registry"]
```

Comment:
```suggestion
[id="ocp-4-15-openshift-default-registry"]
```

Reply:
@mjpytlak I actually have no idea how that got in there. It was from stuff I was messing around with yesterday but it shouldn't have saved. Weird. Thanks for pointing it out!